mapper/pojo-base/src/main/java/org/hibernate/search/mapper/pojo/massindexing/MassIndexingMonitor.java

```diff
@@ -4,7 +4,7 @@
  * License: GNU Lesser General Public License (LGPL), version 2.1 or later
  * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
  */
-package org.hibernate.search.mapper.orm.massindexing;
+package org.hibernate.search.mapper.pojo.massindexing;
```

Comment:
It is impossible to extend because it does not allow the style checking, the extender must be named after the module name "HibernateOrm ...", so it does not allow to use the old name "MassIndexingMonitor". I thought about doing "MassIndexingMonitor extends PojoMassIndexingMonitor". But then it requires that the type be named "HibernateOrmMassIndexingMonitor". I just wanted "MassIndexer extends PojoMassIndexer", but the style check does not allow it.

MassIndexer is the only object that is supposed to have differences between mapper-orm and mapper-javabean, there are additional elements not related to the module needs. The rest of the API should be the same. 

To keep old names via extension, either change the style checking conditions or move the mass indexing API to mapper-pojo. The point is not to build separate mass indexes for both modules. Maybe move the latter to mapper-pojo better. Most of the APIs have already moved there.

Reply:
I'll push an update to the style rules, so that you are allowed to do this provided you add `@Deprecated` on the old type.

Reply:
@wklaczynski I updated the style rules and pushed a commit to your branch. You should now be able to do what I suggested, provided you mark the old class as `@Deprecated`.